features/canvas-2d-filter.yml

```diff
@@ -0,0 +1,6 @@
+name: Canvas 2D `filter` property
```

Comment:
Again, I'm skeptical that this is a distinct feature, rather than a later addition to canvas generally. I could be convinced otherwise, but I haven't seen much about this apart from browser vendor bug/issues.

Reply:
The big reason I put it as it's own is the limited support. It's not in Safari, and I didn't want to create the confusion around something that isn't supported across browsers.